tests/test_mailtrap_webhooks.py

```diff
@@ -0,0 +1,375 @@
+import json
+from datetime import datetime, timezone
+from unittest.mock import ANY
+
+from django.test import tag
+
+from anymail.signals import AnymailTrackingEvent
+from anymail.webhooks.mailtrap import MailtrapTrackingWebhookView
+
+from .webhook_cases import WebhookBasicAuthTestCase, WebhookTestCase
+
+
+@tag("mailtrap")
+class MailtrapWebhookSecurityTestCase(WebhookBasicAuthTestCase):
+    def call_webhook(self):
+        return self.client.post(
+            "/anymail/mailtrap/tracking/",
+            content_type="application/json",
+            data=json.dumps({}),
+        )
```

Comment:
When content_type is application/json, Django's test Client.post will handle json serialization for you:

```suggestion
        return self.client.post(
            "/anymail/mailtrap/tracking/",
            content_type="application/json",
            data={},
        )
```

(This was added in Django 2.1, but some of Anymail's older tests haven't been updated to remove the unnecessary json.dumps().)